src/icalendar/param.py

```diff
@@ -476,6 +485,243 @@ def _default_return_parent() -> enums.RELTYPE:
+
+        LINK;FMTTYPE=text/html;LINKREL=SOURCE;LABEL=Venue;VALUE=URI:
+         https://example.com/venue
+
+    """,
+)
+
+VALUE = string_parameter(
+    "VALUE",
+    """VALUE explicitly specifies the value type format for a property value.
+
+Conformance:
+    VALUE is specified in :rfc:`5545`.
+
+Description:
+    This parameter specifies the value type and format of
+    the property value.  The property values MUST be of a single value
+    type.  For example, a "RDATE" property cannot have a combination
+    of DATE-TIME and TIME value types.
+
+    If the property's value is the default value type, then this
+    parameter need not be specified.  However, if the property's
+    default value type is overridden by some other allowable value
+    type, then this parameter MUST be specified.
+
+    Applications MUST preserve the value data for x-name and iana-
+    token values that they don't recognize without attempting to
+    interpret or parse the value data.
+
+""",
+)
+
+LINKREL = string_parameter(
+    "LINKREL",
+    """LINKREL
+
+Purpose:
+    LINKREL specifies the relationship of data referenced
+    by a LINK property.
+
+Conformance:
+    LINKREL is specified in :rfc:`9253`.
+    This parameter maps to the link relation type defined in
+    Section 2.1 of :rfc:`8288`.
+    It is always quoted.
+
+Description:
+    This parameter MUST be specified on all LINK properties and define
+    the type of reference.
+    This allows programs consuming this data to automatically scan
+    for references they support.
+    There is no default relation type.Any link relation in the
+    link registry established by :rfc:`8288`, or new link relations,
+    may be used.
+    It is expected that link relation types seeing significant usage
+    in calendaring will have the calendaring usage described in an RFC.
+
+    In the simplest case, a link relation type identifies the semantics
+    of a link.  For example, a link with the relation type "copyright"
+    indicates that the current link context has a copyright resource at
+    the link target.
+
+    Link relation types can also be used to indicate that the target
+    resource has particular attributes, or exhibits particular
+    behaviours; for example, a "service" link implies that the link
+    target can be used as part of a defined protocol (in this case, a
+    service description).
+
+Registration:
+    There are two kinds of relation types: registered and extension.
+    These relation types are registered in :rfc:`8288`.
+
+.. seealso::
+
+    `Registered Link Relation Types
+    <https://www.iana.org/assignments/link-relations/link-relations.xhtml>`_.
+
+
+Examples:
+    This identifies the latest version of the event information.
+
+    .. code-block:: text
+
+        LINKREL=latest-version
+
+""",
+)
+
+
+def _get_GAP(prop) -> timedelta | None:  # noqa: N802
+    """GAP
+
+    Purpose:
+        GAP pecifies the length of the gap, positive or negative,
```

Comment:
```suggestion
        GAP specifies the length of the gap, positive or negative,
```

Missing 's' in "specifies".